src/select-list-view.ts

```diff
@@ -1,9 +1,30 @@
-const {Disposable, CompositeDisposable, TextEditor} = require('atom')
+// TODO: fix redundant returns in methods
+
 const etch = require('etch')
```

Comment:
Can this also be converted into an import for consistency? It would be either `import etch from 'etch'` or `import * as etch from 'etch'`, right?

Reply:
Fixed in https://github.com/atom/atom-select-list/pull/28/commits/eaf175ddfce7bc0c96ab5c5324d3be3c991efa98